azerothcore/azerothcore-wotlk
# fix(DB/gameobject): improve Captain Kelisendra's Cargo spawnpoints

## Changes Proposed:
-  New spawns, pooled, under 30 second timer  

## SOURCE:

Wotlk Classic Sniffs

## Tests Performed:

- Run locally, looks to work fine.


 ## How to Test the Changes:

(As Horde)
1.  .tele goldenstrand 
2. .quest add 8886
3. check out some of the new spawns

## Known Issues and TODO List:


- Part 2 of what will be at least 3 parts on this area.  Split because I wanted to get as much submitted before I went to work as possible, got this done right out the door :P 


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
